.github/workflows/erlang.yml

```diff
@@ -26,3 +26,51 @@ jobs:
       run: rebar3 xref
     - name: Run elvis
       run: rebar3 as lint lint
+  validate:
+    name: Validate PR title
+    runs-on: ubuntu-latest
+    steps:
+        - uses: amannn/action-semantic-pull-request@v5
+          env:
+            GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          with:
+            types: |
+              chore
+              feat
+              fix
+              style
+              test
+        - name: Get Next Version
+          if: github.ref == 'refs/heads/master'
+          env:
+            GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          id: semver
+          uses: ietf-tools/semver-action@v1
+          with:
+            token: ${{ github.token }}
+            branch: main
```

Comment:
Shouldn't it be `master` here instead of ´main`?

Reply:
Yes, should be master